hera_cal/io.py

```diff
@@ -864,6 +864,147 @@ def iterate_over_times(self, Nints=1, times=None):
             yield self.read(times=times[i:i + Nints])
 
 
+def read_hera_hdf5(filenames, bls=None, pols=None, full_read_thresh=0.002,
+                   read_data=True, read_flags=False, read_nsamples=False, check=False,
+                   dtype=np.complex128, verbose=False):
+    '''A ~100x faster interface to getting data out of HERA HDF5 files. Only concatenates
+    along time axis. Assumes that if antenna_numbers header array stays the same,
```

Comment:
`antenna_numbers` is out of date, you look at `ant_1_array` and `ant_2_array now, right?

Reply:
Removed